pkg/apis/componentconfig/types.go

```diff
@@ -142,6 +142,9 @@ type KubeletConfiguration struct {
 	// podInfraContainerImage is the image whose network/ipc namespaces
 	// containers in each pod will use.
 	PodInfraContainerImage string `json:"podInfraContainerImage"`
+	// podInfraContainerImagePullSecret is the path to a json or yaml file
```

Comment:
Add a `File` suffix to make it explicit. 
Not for this PR: I wish we can consume the contents of the file instead of the filename here.